### PR TITLE
feat: add sitemap url for de instance

### PR DESCRIPTION
**Pull request by elbotho**
(do not deply to production until frontend is done)

### CODE REVIEW

**Review by kulla — APPROVED** [review]
Feel free to merge, when needed

File: src/robots.ts [hunk 1]
```diff
@@ -22,9 +22,15 @@ export function robotsTxt(request: Request, env: CFEnvironment) {
...
   if (url.pathname !== '/robots.txt') return null
 
+  const sitemap =
+    isInstance(url.subdomain) && url.subdomain === Instance.De
+      ? `
+  Sitemap: https://de.serlo.org/sitemap.xml`
+      : ''
+
   return new Response(
     env.ENVIRONMENT === 'production'
```
Comment (shn-srl) [inline src/robots.ts:34]: nit: could be nice to define this in a const and have a cleaner return line

File: src/robots.ts [hunk 2]
```diff
@@ -22,9 +22,15 @@ export function robotsTxt(request: Request, env: CFEnvironment) {
   const url = Url.fromRequest(request)
   if (url.pathname !== '/robots.txt') return null
 
+  const sitemap =
+    isInstance(url.subdomain) && url.subdomain === Instance.De
```
Comment (shn-srl) [inline src/robots.ts:29]: nit: also nice for the boolean to be declared as a well named variable, then the ternary logic is super clear

**Review by shn-srl — APPROVED** [review]

**Comment by elbotho** [comment]
@shn-srl thanks for the feedback

**Pull request merged**